src/modules/px4iofirmware/i2c.c

```diff
@@ -77,14 +79,12 @@ static void		i2c_dump(void);
 static DMA_HANDLE	rx_dma;
 static DMA_HANDLE	tx_dma;
 
-static uint8_t		rx_buf[68];
+static uint8_t		xfer_buf[72];
 static unsigned		rx_len;
 
-static const uint8_t	junk_buf[] = { 0xff, 0xff, 0xff, 0xff };
```

Comment:
..or the architectural change to reuse the RX buf instead of the "junk but" for TX had some side effects. Apparently the RX but was increased in size exactly by the size of the TX but, but its unclear to me where the CRC fits in - both buffers should have grown.